bot.user.js

```diff
@@ -379,34 +385,24 @@ window.checkCollision = function(x, y, r) {
     if (window.visualDebugging) {
         window.drawDot(circle1.x, circle1.y, circle1.radius, 'blue', false);
     }
-    var shortest_distance = 10000;
     var avoid = false;
-    var circle2;
-
-    for (var snake in window.snakes) {
-        if (window.snakes[snake].nk != window.snake.nk) {
-            for (var y = window.snakes[snake].pts.length - 1; 0 <= y; y--) {
-                if (!window.snakes[snake].pts[y].dying) {
-                    var xx = window.snakes[snake].pts[y].xx + window.snakes[snake].fx;
-                    var yy = window.snakes[snake].pts[y].yy + window.snakes[snake].fy;
-                    circle2 = {
-                        x: xx,
-                        y: yy,
-                        radius: 15 * window.snakes[snake].sc * window.getScale()
-                    };
-                    if (window.circleIntersect(circle1, collisionScreenToCanvas(circle2))) {
-                        var distance = window.getDistance(window.getX(), window.getY(), xx, yy);
-                        if (distance < shortest_distance){
-                            window.changeGoalCoords(circle2);
-                            avoid = true;
-                            shortest_distance = distance;
-                        }
-                    }
-                }
-            }
-        }
-    }
-    return avoid;
+	if (window.collisionPoints[0] != null){
+		var circle2;
+		var multiplier = 1;
```

Comment:
Seems like the 'multiplier' variable is not really used for anything,  at least not anymore.